src/app/landing/landing.component.scss

```
@@ -0,0 +1,201 @@
+  font-weight: 500;
+  color: mat.m2-get-color-from-palette(app.$app-primary, "text");
+}
+
+.landing-logo .logo {
+  display: flex;
+  align-items: center;
+  color: inherit;
+}
+
+.landing-logo .logo svg {
+  fill: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)